lib/r10k/forge/module_release.rb

```diff
@@ -13,6 +13,7 @@ class ModuleRelease
 
       def_setting_attr :proxy
       def_setting_attr :baseurl
+      def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/cache': '/root/.r10k/cache')
```

Comment:
i don't have context on why the other cache was in root to begin with, so it's probably fine, just struck me as odd.